tools/legal-review-helper/package-lock.json

```diff
@@ -0,0 +1,645 @@
+{
```

Comment:
I'm not sure if we should commit the package lock for this tool.

Reply:
I'm not sure, toplevel has package-lock committed. I guess there's no harm in it being here too. If it's a problem we can remove it.